### PR TITLE
bump draft version to v0.16.0

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]

File: Formula/draft.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Draft < Formula
   desc "A tool for developers to create cloud-native applications on Kubernetes."
   homepage "https://github.com/Azure/draft"
-  version "0.15.0"
+  version "0.16.0"
   url "https://azuredraft.blob.core.windows.net/draft/draft-v#{version}-darwin-amd64.tar.gz"
   sha256 "c1b3da0d2dd3e86bbd4aa259c91710bdf94f711d37ff98ba4279977c22b90fbe"
```
Comment (bacongobbler) [inline Formula/draft.rb]: Can you change the shasum?

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by michelleN** [comment]
@bacongobbler done. good catch!

**Review by bacongobbler — APPROVED** [review]

**Comment by harmjanblok** [comment]
Any plans to get this merged?

FYI: workaround until this has been merged:

```
brew reinstall https://raw.githubusercontent.com/Azure/homebrew-draft/b55619491da20d95887cb7237afcdbe07707bc14/Formula/draft.rb
```

**Comment by michelleN** [comment]
Thanks @harmjanblok - just merged!

**Pull request merged**